airgun/views/all_hosts.py

```diff
@@ -347,3 +347,106 @@ class review(WizardStepView):
+    cancel_btn = Button(locator='//button[normalize-space(.)="Cancel"]')
+    back_btn = Button(locator='//button[normalize-space(.)="Back"]')
+    next_btn = Button(locator='//button[normalize-space(.)="Next"]')
+
+    @View.nested
+    class select_errata(WizardStepView):
+        wizard_step_name = "Select errata"
+        locator_prefix = f'.//div[contains(., "{wizard_step_name}")]/descendant::'
+        expander = Text(f'.//button[text()="{wizard_step_name}"]')
+        content_text = Text('.//div[@class="pf-c-content"]')
+
+        select_all = Checkbox(locator=f'{locator_prefix}div[@id="selection-checkbox"]')
+        search_input = SearchInput(locator=f'{locator_prefix}input[@aria-label="Search input"]')
+        clear_search = Button(locator=f'{locator_prefix}button[@aria-label="Reset search"]')
+        search = Button(locator=f'{locator_prefix}button[@aria-label="Search"]')
+
+        table = PatternflyTable(
+            component_id='table',
+            column_widgets={
+                0: Checkbox(locator='.//input[@type="checkbox"]'),
+                'Erratum': Text('.//td[2]'),
+                'Title': Text('.//td[3]'),
+                'Type': Text('.//td[4]'),
+                'Severity': Text('.//td[5]'),
+                'Affected hosts': Text('.//td[6]'),
+            },
+        )
+        pagination = Pagination()
+
+    @View.nested
+    class review_hosts(WizardStepView):
+        wizard_step_name = "Review hosts"
+        locator_prefix = f'.//div[contains(., "{wizard_step_name}")]/descendant::'
+
+        expander = Text(f'.//button[text()="{wizard_step_name}"]')
+        content_text = Text('.//div[@class="pf-c-content"]')
+        error_message = OUIAAlert('no-hosts-alert')
+
+        select_all = Checkbox(locator=f'{locator_prefix}div[@id="selection-checkbox"]')
+        search_input = SearchInput(locator=f'{locator_prefix}input[@aria-label="Search input"]')
+        search = Button(locator=f'{locator_prefix}button[@aria-label="Search"]')
+
+        table = PatternflyTable(
+            component_id='table',
+            column_widgets={
+                0: Checkbox(locator='.//input[@type="checkbox"]'),
+                'Name': Text('.//td[2]'),
+                'OS': Text('.//td[3]'),
+            },
+        )
+        pagination = Pagination()
+
+    @View.nested
+    class review(WizardStepView):
+        wizard_step_name = "Review"
+        expander = Text(f'.//button[text()="{wizard_step_name}"]')
+
+        tree_expander_errata = Button(
+            './/button[@class="pf-c-tree-view__node" and contains(.,"Errata to")]'
+        )
+        expanded_errata_list = ItemsList(
+            locator='//ul[@class="pf-c-tree-view__list" and @role="tree"][1]'
+        )
+
+        number_of_errata_to_manage = Text(
+            '''
+                                            .//span[contains(.,"Errata to")]/following-sibling::
```

Comment:
Is there a reason the spacing is like this?

Reply:
yeah idk kinda gross
it's fixed now